src/jobs/push-to-app-collection.yaml

```diff
@@ -0,0 +1,19 @@
+executor: architect
+parameters:
+  app_name:
+    description: "Name of the application"
+    type: "string"
+  app_catalog:
+    description: "Name of the catalog CR where the application belongs"
+    type: "string"
+    default: "control-plane-catalog"
+  app_collection_repo:
+    description: "Name of the app collection repository"
+    type: "string"
+steps:
+  - checkout
+  - prepare-catalogbot-git-ssh
+  - push-to-app-collection:
+      app_name: <<parameters.app_name>>
+      app_catalog: <<parameters.app_catalog>>
```

Comment:
I guess we don't have to worry about test catalog here?

Reply:
We should not, but it felt strange to hardcode the catalog name also.